walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -28,6 +31,36 @@ public void testGreeting() {
 
     String greeting = greeter.greet("Ada");
 
-    Assert.assertEquals("Hello Ada", greeting);
+    assertThat(greeting).isEqualTo("Hello Ada");
+  }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
+
+    String greeting = greeter.greet("   Ada   ");
+
+    // Whitespace should be trimmed
```

Comment:
No need for the comment here - the test method name already says the same.

(Google Style discourages "overuse" of comments; experience shows that engineers tend to evolve the code and forget to update comments accordingly.)